### PR TITLE
adds metrics

### PR DESCRIPTION
**Pull request by sandikbarr**
adds metrics?

### CODE REVIEW

File: stacks/root.yml [hunk 1]
```diff
@@ -116,6 +120,8 @@ Mappings:
       prefix: 38
     Crier:
       prefix: 42
+    Metrics:
```
Comment (gcampo88) [inline stacks/root.yml]: Metrics should have a unique # here (jingle has already taken 38). Also good to maintain order in increasing listener priority. My guess is Metrics should be somewhere around -- 36? Lower priority than castle (which will get hit at least as often as metrics) but higher than jingle (which will only be the shows we sell ads for)

Comment (sandikbarr) [reply stacks/root.yml]: done!

Comment (gcampo88) [reply stacks/root.yml]: last nitpicky thing -- good to maintain this list in order of the priority prefix, so just move this to below castle / above jingle and then i think all set!

**Comment by gcampo88** [comment]
LGTM! Will hold off on merging til we're ready to deal with the consequences :). 🥇 💯 📊

**Pull request merged**